src/tests/Microsoft.Diagnostics.NETCore.Client/CommonHelper.cs

```diff
@@ -2,13 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.IO;
 using System.Runtime.InteropServices;
 
 namespace Microsoft.Diagnostics.NETCore.Client
 {
     public class CommonHelper
     {
+        // This is a stopgap until Arcade updates to an RC version of the SDK.
+        // KEEP IN SYNC WITH eng/Versions.xml
+        public static readonly string CurrentDARCVersion = "5.0.0-rc.1.20425.1";
```

Comment:
that's a little crazy :O

Reply:
I'm pondering an automated way of doing this.  Best I've come up with is adding some logic to build.{sh|cmd} that injects this value here before we build.